demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -86,14 +86,42 @@ def get_playbook_dependencies(playbook):
     playbooks = set()
 
     playbook_tasks = playbook.get('tasks')
+    playbook_path = os.path.relpath(input)
+    pack_path = os.path.dirname(os.path.dirname(playbook_path))
+    integration_dir_path = os.path.join(pack_path, 'Integrations')
+    # Get all files in integrations directories
+    pack_files = [os.path.join(r, file) for r, d, f in os.walk(integration_dir_path) for file in f]
```

Comment:
looks like we have util func for that named `get_yml_paths_in_dir`
see if you can use that here